resources/dicts/patrols/beach/hunting/leaf-fall.json

```diff
@@ -358,19 +358,18 @@
     "p_l knows of a spot where they can shelter in between some bushes and wait out the rain. r_c suddenly inhales sharply and directs their attention to a small nest full of baby birds who have fallen from a tree because of the wind. They are wet but happy, now they can return to camp with their heads held high and rest happily and hopefully warm in their nests later that day knowing they have fed their Clan."
     ],
     "fail_text": [
-    "p_l and r_c end up arguing about whether or not to continue the patrol, the rest of the cats are not happy with them both after they decide to go back to camp after already being soaked to the bone, no prey and still wet what a waste of time.",
-    null,
-    "p_l and the rest of the patrol decide to seek shelter under some trees. r_c sees a large tree with a weird, flat object attached to it. They race ahead, happy at the prospect of getting out of the rain. As they run under the tree they suddenly scream out in pain, the rest of the patrol can only watch in terror as r_c falls and spasms beneath the tree, the cause seeming to be the rain dripping from it's leaves. p_l swallows their horror and orders their patrol back to camp, they'll have to inform everyone of this tree and discuss if its worth it to recover r_c's body, they don't want any other cat to die such a horrifying death.",
-    "p_l sees a large tree and heads towards it for shelter. Just as the patrol comes under it's leaves, r_c yelps and leaps away! The rain dripping from the tree's leaves seems to burn them and the patrol quickly runs back out into the downpour. The patrol decides to head back to camp, both needing to warn the Clan about this deceptive tree and needing to treat r_c's new wounds."
+    "p_l and r_c end up arguing about whether or not to continue the patrol, the rest of the cats are not happy with both of them after they decide to go back to camp after already being soaked to the bone, no prey and still wet what a waste of time.",
+     null,
+    "The patrol decides to take shelter. app1 sees a large tree and makes a run for it not hearing the screams from p_l to come back. Their mistake dawns on them the moment they take shelter underneath what they now realize is the death tree but they are too late to escape the poison rain now pouring down on their body, they let out a scream before falling still in a spasm. The patrol returns to c_n both eyes and fur wet.",
```

Comment:
A few grammar issues, could be fixed to:
"The patrol decides to take shelter. app1 sees a large tree and makes a run for it, not hearing **p_l's scream of warning**. _Their mistake dawns on them the moment they take shelter underneath what they now realize is the death tree but they are too late to escape the poison rain now pouring down on their body, they let out a scream before falling still in a spasm._ (_this whole sentence feels a bit run-on and confusing, could try rewriting it or breaking it into multiple sentences_) The patrol returns to c_n, both eyes and fur wet."